benchmarks/retrieval/retrieve.py

```diff
@@ -83,13 +84,18 @@ def main():
         retrieved = retriever.invoke(item[args.question_field])
         item["retrieved"] = []
         for doc_idx, doc in enumerate(retrieved):
+            pprint.pprint(doc)
```

Comment:
These documents can get extremely long. Could you just use logging.info() here, and then set the appropriate logging level whenever you want to see the documents?

Reply:
stray print. removing